WtWSMS_EBS/history/characters/roman.txt

```diff
@@ -1224,7 +1230,23 @@
 		death="240.1.1"
 	}
 }
-
+145252 = {
```

Comment:
ID duplicate with a WtWSMS character, which is Claudius too.

Reply:
fixed